logger.py

```diff
@@ -30,8 +31,9 @@ def log(self, msg: str = '', level: str = 'INFO') -> None:
         fulltext = color + pfx + str(msg) + reset + os.linesep
         sys.stderr.write(fulltext)
         sys.stderr.flush()
-        with open('/tmp/test', 'a', 1, encoding='utf-8') as testfile:
-            testfile.write(logtext)
+
+        with Path('/tmp/test').open(mode='a', encoding='utf-8', buffering=1) as file:
+            file.write(logtext)
```

Comment:
We can actually refactor this. But putting that aside why do we need to test if /tmp is writable? We should delete this and prefer adding this as part of our checks.

Reply:
Never mind. We’re not testing anything here. In those lines,  we’re writing the log message. 

TODO: Refactor to create a secure temporary file.